core/integration/engine_test.go

```diff
@@ -50,15 +57,12 @@ func devEngineContainer(c *dagger.Client, withs ...func(*dagger.Container) *dagg
 	return ctr.
 		WithMountedCache("/var/lib/dagger", c.CacheVolume("dagger-dev-engine-state-"+identity.NewID())).
 		WithExposedPort(1234, dagger.ContainerWithExposedPortOpts{Protocol: dagger.NetworkProtocolTcp}).
-		WithExec([]string{
+		WithDefaultArgs([]string{
 			"--addr", "tcp://0.0.0.0:1234",
```

Comment:
What's the actual entrypoint? Can it be moved as first argument here?

Reply:
I think its coming from the OCI import of `engine tar file`. I can add it here explicitly, but I am thinking it may be a. duplicate call here. what do you think?

Reply:
If you can track where that's defined and used, I'd probably remove the entrypoint and explicitly add the argument on the callers.

Reply:
I have moved the entrypoint config to the caller. It seems like that is the only place we are using this from code perspective, but i am also wondering if something implicitly depends on that. e.g. `hack/dev`?

Reply:
It seems like this triggered failures in other areas. I just removed this commit to verify if that fixes the CI issues.